src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -1001,6 +1030,18 @@ private void enableOrDisableFields() {
 		bossBarType.setEnabled(isBoss.isSelected());
 
 		rangedAttackItem.setEnabled("Default item".equals(rangedItemType.getSelectedItem()));
+
+		celebrationSound.setEnabled(mobBehaviourType.getSelectedItem() == "Raider");

```

Comment:
Please read https://stackoverflow.com/questions/513832/how-do-i-compare-strings-in-java and improve this. Also make tmp boolean isRaider so you don't do compare so many times

Reply:
Ok

Reply:
Done